### PR TITLE
xmrig: Add version 5.11.3

### PR DESCRIPTION
**Pull request by steffentbm**

### CODE REVIEW

File: bucket/xmrig.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "checkver": {
+        "github": "https://github.com/xmrig/xmrig",
+        "regex": "xmrig-(\\d+\\.\\d+\\.\\d+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/xmrig/xmrig/releases/download/v$version/xmrig-$version-msvc-win64.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/xmrig.json:26]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/SHA256SUMS"
        },
        "extract_dir": "xmrig-$version"
```

File: bucket/xmrig.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "extract_dir": "xmrig",
+    "bin": [
+        "xmrig.exe",
+        "xmrig-notls.exe"
+    ],
+    "checkver": {
+        "github": "https://github.com/xmrig/xmrig",
+        "regex": "xmrig-(\\d+\\.\\d+\\.\\d+)"
```
Comment (Ash258) [inline bucket/xmrig.json:19]:
```suggestion
        "github": "https://github.com/xmrig/xmrig"
```

File: bucket/xmrig.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "url": "https://github.com/xmrig/xmrig/releases/download/v5.11.3/xmrig-5.11.3-msvc-win64.zip",
+            "hash": "968b06d007f218f8e3c882b3b6d76293fa2533f282bcffc9b727c36ca31e5db5"
+        }
+    },
+    "extract_dir": "xmrig",
+    "bin": [
+        "xmrig.exe",
+        "xmrig-notls.exe"
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/xmrig.json:17]:
```suggestion
    "persist": "config.json",
    "checkver": {
```

File: bucket/xmrig.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "version": "5.11.3",
+    "description": "XMRig High performance, open source, cross platform RandomX, CryptoNight, AstroBWT and Argon2 CPU/GPU miner, with official support for Windows.",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xmrig/xmrig/releases/download/v5.11.3/xmrig-5.11.3-msvc-win64.zip",
+            "hash": "968b06d007f218f8e3c882b3b6d76293fa2533f282bcffc9b727c36ca31e5db5"
+        }
+    },
+    "extract_dir": "xmrig",
```
Comment (Ash258) [inline bucket/xmrig.json:12]:
```suggestion
    "extract_dir": "xmrig-5.11.3",
```

File: bucket/xmrig.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://xmrig.com/",
+    "version": "5.11.3",
+    "description": "XMRig High performance, open source, cross platform RandomX, CryptoNight, AstroBWT and Argon2 CPU/GPU miner, with official support for Windows.",
```
Comment (Ash258) [inline bucket/xmrig.json:4]:
```suggestion
    "version": "5.11.3",
    "description": "High performance RandomX, CryptoNight, AstroBWT and Argon2 CPU/GPU miner",
    "homepage": "https://xmrig.com",
```

**Comment by steffentbm** [comment]
How do I proceed now?

**Pull request merged**